environments/devlive.env

```diff
@@ -16,8 +16,8 @@ IPFS_EMULATOR=8378
 
 # Default IPFS access key.  These values must match the
 # the token secret passed to mock-ipfs-pinning-service
-PINATA_API_KEY=ipfskey
-PINATA_API_SECRET=ipfspwd
+PINATA_API_KEY=36d74aea0159902a10df
+PINATA_API_SECRET=9fd41541e589b957756850b18098e459fedfe29439e5f6a888f65ace5c65ae6d
```

Comment:
faaarrrrr....

Put this in (just while testing) here because we (currently) don't have a mock for IPFS.  Thats what I get for cutting corners

Reverted in the next PR, and keys revoked,